src/content/config.ts

```diff
@@ -12,6 +12,23 @@ const sponsorsCollection = defineCollection({
     }),
   ),
 });
+
+const staffCollection = defineCollection({
+  type: "data",
+  schema: z.array(
+    z.object({
+      id: z.number(),
+      name: z.string(),
+      icon: z.string(),
```

Comment:
こちらもおそらくアセットのURLを登録していくことになると思うので、URLでお願いします！
好みですがキー名はサムネイルでも良いかなと思いました 🙆‍♂️ 
```suggestion
     thumbnail: z.string().url(),
```